src/main/java/core/basesyntax/ColorEnum.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum ColorEnum {

```

Comment:
Let's rename it simply to `Color`. `enum` keyword already said that it is an enum.